deployment/cfn/application.py

```diff
@@ -83,7 +83,7 @@ class Application(StackNode):
         'StackType': 'Staging',
         'StackColor': 'Green',
         'KeyName': 'mmw-stg',
-        'AppServerInstanceType': 't2.micro',
+        'AppServerInstanceType': 't2.small',
```

Comment:
These are OK to change to keep things in sync, but they are effectively treated as defaults in the event that `AppServerInstanceType` isn't defined anywhere up the chain. I think a corresponding change in `default.yml.example` would complete the syncing of possible defaults.

That said, the persistence of this is driven by a local copy of `default.yml` on Jenkins (for staging) and the file share (for production). Changes would need to be made to those for future deployments to use a `t2.small`. (We can do this for staging now and test an out-of-band deployment via Jenkins to confirm it works.)

Lastly, the change from a `t2.micro` -> `t2.small` makes us come up short based on our existing EC2 reservations. Each `t2.micro` equates to 2 `t2.nano` and each `t2.small` equates to 4 `t2.nano`, so I think that we'd need to purchase 4 `t2.nano` units to have full coverage again.

We'd probably want to get signoff on that before doing it since it ends up being a  1 year commitment.

See: https://aws.amazon.com/blogs/aws/new-instance-size-flexibility-for-ec2-reserved-instances/

Reply:
`t2.small` application server is up on staging now.

Reply:
Sounds good. I'll hold off on purchasing until we get a confirmation from Stroud.